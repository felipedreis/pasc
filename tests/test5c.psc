var j, k, result is int;
begin
 k := k - j;
 in(j);
 in(k);

 if k = 0 then
    result := j/k;
 end;

 out(result);
end