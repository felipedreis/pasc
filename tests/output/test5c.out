Opening file tests/test5c.psc...
compilation finished with success
